internal/storage/connector.go

```diff
@@ -48,12 +48,9 @@ type IStagingStorage interface {
 }
 
 type IMainStorage interface {
-	InsertBlocks(blocks []common.Block) error
-	InsertTransactions(txs []common.Transaction) error
-	InsertLogs(logs []common.Log) error
-	InsertTraces(traces []common.Trace) error
+	InsertDataForBlocks(data []common.BlockData) error
```

Comment:
nit: `InsertBlockData` is maybe a better name for this function

Reply:
Yeah, but it clashes with Staging interface where there is also a `InsertBlockData`. Do you have other naming suggestions for these 2?

Reply:
in staging we don't insert blocks anymore, we can change the staging function to `InsertStaginData`?